SOURCE_OV_WORKSPACE

```diff
@@ -0,0 +1,8 @@
+# OPENVINO DEFINITION FOR BUILDING FROM SOURCE:
+
+new_local_repository(
+    name = "openvino",
+    build_file = "@//third_party/openvino:BUILD",
```

Comment:
Wasn't it easier to keep a single WORKSPACE file?
The definition of build_file/build_file_content is the same.
All you need to do is to keep the same path. For example, keep the definition for binary version (inference_engine/lib/intel64/libinference_engine_legacy.so) and in case of compiling from sources create a symbolic link from /openvino/bin/intel64/Release/lib/ to /opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/
The second symbolic link is for includes.
That way, we'll have a single WORKSPACE file with no special hacks in Makefile